client/src/components/History/Content/model/states.js

```diff
@@ -19,12 +19,17 @@ export const STATES = {
         status: "success",
         text: "No data.",
     },
-    /** the tool producing this dataset failed */
+    /** the tool producing this dataset has errored */
     error: {
         status: "danger",
         text: "An error occurred with this dataset:",
         icon: "exclamation-triangle",
     },
+    /** the dataset has failed */
```

Comment:
```suggestion
    /** the job has failed */
```

Reply:
I'm okay with that, but we should at least leave a hint that this state is not like the others